tests/steps/conftest.py

```diff
@@ -17,7 +18,7 @@ def scenario_context():
 def browser():
     chrome_options = Options()
     from tests.config import RUN_HEADLESS_UI_TESTS
-    if RUN_HEADLESS_UI_TESTS:
+    if RUN_HEADLESS_UI_TESTS is True or RUN_HEADLESS_UI_TESTS == 'True':
```

Comment:
a less verbose version would be:
```suggestion
    if RUN_HEADLESS_UI_TESTS in (True, 'True'):
```

Reply:
We should make sure the variable is always of boolean type.

` 
if RUN_HEADLESS_UI_TESTS:
`